src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -633,6 +633,16 @@ def check_organization_by_slug(self, *, slug: str, only_visible: bool) -> Option
 
         return None
 
+    def check_organization_by_id(self, *, id: int, only_visible: bool) -> bool:
+        # See RegionOrganizationCheckService below
+        org_mapping = OrganizationMapping.objects.filter(organization_id=id).first()
+        if org_mapping is None:
+            logger.info(f"Organization by id not found: {id}")
```

Comment:
```suggestion
            logger.info(f"Organization by id not found", extra={"id": id})
```
We should avoid formatting values into log messages as it makes aggregation harder.